src/app/components/Modal/Modal.tsx

```diff
@@ -18,14 +19,33 @@ type ModalContentProps = {
 	title: string;
 	description?: string;
 	image?: React.ReactNode;
+	size?: "small" | "medium" | "large" | "xlarge" | "default";
 	onClose?: any;
 };
 
+const ModalContainer = styled.div<{ size: string }>`
+	${({ size }) => {
+		switch (size) {
+			case "small":
```

Comment:
Maybe we should stick to the same naming as the classes itself. Meaning `sm/md/lg/xl` because that is what people already know from Tailwind.

Reply:
Yeah, I agree

Reply:
There seem to be a few other components where the same naming conventions could be applied.